### PR TITLE
doc new manifest elements and update toc

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
This fixes internal item OC 3468497
The TOC changes in include adding EqivalentAddins and EquivalentAddin. Those articles were already published. They just weren't in the TOC.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f2dab81](https://github.com/OfficeDev/office-js-docs-pr/commits/f2dab81820753460de4837de014c996ce1ea4566)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/authorization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/authorization.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/authorizations.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/authorizations.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/scopes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/scopes.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/versionoverrides.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/versionoverrides.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/webapplicationinfo.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/webapplicationinfo.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201908141900295770-1249%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=og8n5E0GcE3TwsdITzJtjFgBGYiYD4fCw4CWrlHRKkI%3D&st=2019-08-14T18%3A56%3A42Z&se=2019-09-14T19%3A01%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/manifest/authorization.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## Example
+
+```xml
+<OfficeApp>
+...
+  <VersionOverrides xmlns="http://schemas.microsoft.com/office/mailappversionoverrides" xsi:type="VersionOverridesV1_0">
+    ...
+    <WebApplicationInfo>
+      <Id>12345678-abcd-1234-efab-123456789abc</Id>
+      <Resource>api://myDomain.com/12345678-abcd-1234-efab-123456789abc</Resource>
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/manifest/authorization.md]: Do we need to note/comment that myDomain and contoso are placeholder domains?

File: docs/reference/manifest/versionoverrides.md [hunk 2]
```diff
@@ -28,8 +28,9 @@ The root element that contains information for the add-in commands implemented b
 |  **Requirements**  |  No   |  Specifies the minimum requirement set and version of Office.js that the add-in requires. This overrides the  `Requirements` element in the parent portion of the manifest.|
 |  [Hosts](./hosts.md)                |  Yes  |  Specifies a collection of Office hosts. The child  Hosts element overrides the Hosts element in the parent portion of the manifest.  |
 |  [Resources](./resources.md)    |  Yes  | Defines a collection of resources (strings, URLs, and images) that other manifest elements reference.|
+|  [EquivalentAddins](./equivalentaddins.md)    |  No  | Specifies the native (COM/XLL) add-ins that are equivalent to the web add-in. The web add-in is not activated in an equivalent native add-in is installed.|
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/manifest/versionoverrides.md]: if?
```suggestion
|  [EquivalentAddins](./equivalentaddins.md)    |  No  | Specifies the native (COM/XLL) add-ins that are equivalent to the web add-in. The web add-in is not activated if an equivalent native add-in is installed.|
```

File: docs/reference/manifest/versionoverrides.md [hunk 3]
```diff
@@ -28,8 +28,9 @@ The root element that contains information for the add-in commands implemented b
...
 |  **VersionOverrides**    |  No  | Defines add-in commands under a newer schema version. See [Implementing multiple versions](#implementing-multiple-versions) for details. |
-|  **WebApplicationInfo**    |  No  | Specifies details about the add-in's associated Web application. |
+|  [WebApplicationInfo](./webapplicationinfo.md)    |  No  | Specifies details about the add-in's registration with secure token issues, such as Azure Active Directory V2.0. |
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/manifest/versionoverrides.md]:
```suggestion
|  [WebApplicationInfo](./webapplicationinfo.md)    |  No  | Specifies details about the add-in's registration with secure token issuers, such as Azure Active Directory V2.0. |
```

File: docs/reference/manifest/authorization.md (hunk 1 above)
Comment (Rick-Kirkham) [reply docs/reference/manifest/authorization.md]: No. I don't think that's needed.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a16a5bb](https://github.com/OfficeDev/office-js-docs-pr/commits/a16a5bbe92846bb360a4b02a2c409cc1dc224925)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/authorization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/authorization.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/authorizations.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/authorizations.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/scopes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/scopes.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/versionoverrides.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/versionoverrides.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/webapplicationinfo.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/webapplicationinfo.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201908141917489084-1249%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=tT%2BvZDK1iVjG%2BiFP2Au0sY270WftwUj%2FVf8Rrc8hW1Q%3D&st=2019-08-14T19%3A14%3A02Z&se=2019-09-14T19%3A19%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c0bd5a2](https://github.com/OfficeDev/office-js-docs-pr/commits/c0bd5a201894001b883e77d2eb3fbee77c02ec03)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/authorization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/authorization.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/authorizations.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/authorizations.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/scopes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/scopes.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/versionoverrides.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/versionoverrides.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/webapplicationinfo.md](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/reference/manifest/webapplicationinfo.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/newish-manifest-elements/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201908141918137918-1249%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2L5gCnW3CqGSPw%2BmTgaN%2FTviXbMBNitAF55cHRMtrZs%3D&st=2019-08-14T19%3A15%3A17Z&se=2019-09-14T19%3A20%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**